### ISSUE TITLE
Jackett 0.20.2264 fails to build on nixos-22.11

### ISSUE THREAD

**Issue by NorfairKing** [NixOS/nixpkgs#213971]
### Steps To Reproduce

```
nixpkgs @ nixos-22.11 $ nix-build -A jackett
```

### Build log
```
Test Run Failed.
Total tests: 125
     Passed: 123
     Failed: 2
 Total time: 2.7680 Seconds
```

### Notify maintainers

@edwtjo @nyanloutre @purcell 

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.86, NixOS, 22.11 (Raccoon), 22.11.20230115.2f9fd35`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.1`
 - nixpkgs: `/nix/store/d6vlzbp9x6ixk1d1gr7if0f2ji6s8q4v-source`
```

**Comment by pluskal** [comment]
The issue is in the Jackett tests. They depend on the current year.

https://github.com/Jackett/Jackett/commits/master/src/Jackett.Test/Common/Utils/DateTimeUtilTests.cs

**Comment by wegank** [comment]
Closed by #213905.
